libs/libpcf/example/example_custom_pcf/custom_pcf_command.xml

```diff
@@ -15,11 +15,12 @@
       <mode name="0.6ns" value = "110" />
       <mode name="0.7ns" value = "111"/>
     </option>
+    <option name="delay2" type="decimal" num_bits="3" max="6" little_endian="false" offset="0"/>
```

Comment:
In this LINE, the offset should be more than 0. Otherwise, an unexpected bitstream overwritten will occur.